src/psyclone/tests/dynamo0p3_lma_test.py

```diff
@@ -38,9 +38,6 @@
 ''' This module tests the support for LMA operators in the Dynamo 0.3 API
```

Comment:
Would you be so kind to correct this to "LFRic (Dynamo 0.3)"?

Reply:
Done.